Datagrid: update the columns.name option (#1415)
DevExpress/devextreme-documentation: api-reference/_hidden/GridBaseColumn/name.md

---
---
##### shortDescription

> Specifies a unique identifier of the column. The value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField) by default.

Reviewer: ```suggestion
Specifies the column's unique identifier. This value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField).
```

Author: This value is inherited from the [dataField] _only if the name option is not set in code_. Thus can the sentence be 

```suggestion
Specifies the column's unique identifier. If not set in code, this value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField).
```

?

Reviewer: I agree with your suggestion, but your original version "The value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField) by default" and "If not set in code, this value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField)" is different.

> Specifies a unique identifier of the column. If not set in code, the value is autogenerated from [dataField]({basewidgetpath}/Configuration/columns/#dataField).

Reviewer 2: ```suggestion
Specifies a unique identifier of the column. The value is inherited from the [dataField]({basewidgetpath}/Configuration/columns/#dataField) by default.
```

---

> This option's value is used to refer to the column in your code, for example, when [changing a column option]({basewidgetpath}/Methods/#columnOptionid_optionName_optionValue).

Reviewer: ```suggestion
This option's value is used to refer to the column in code, for example, when [changing a column option]({basewidgetpath}/Methods/#columnOptionid_optionName_optionValue).
```
